Greg Block/1.3.1/config/betterquesting/DefaultQuests.json

```
@@ -2961,7 +2961,7 @@
           "autoclaim:1": 0,
           "ismain:1": 1,
           "name:8": "哞~",
-          "desc:8": " It would be a good idea to use the Grass Seeds you got from sifting Dirt to make a small Grass area in middle of your island, where you have a Plains biome. You can then set Baits to spawn specific animals. A Cow is one mob that you will need to progress. You can also quite simply automate collecting Milk from it by placing it on top of a Barrel.[待翻译]"
+          "desc:8": " 最好使用你从泥土中筛出来的草种子，在岛屿中间建一块草地，在那里是一个平原生物群系。然后你可以设置诱饵来生成特定的动物。牛是一种你发展所需要的生物。你还可以将牛放在桶上面实现牛奶的自动收集。You can also quite simply automate collecting Milk from it by placing it on top of a Barrel.（此处测试中无法实现，故保留原文。）"
```

```suggestion
          "desc:8": " 最好使用你从泥土中筛出来的草种子，在岛屿中间建一块草地。那里是平原生物群系，然后你就可以设置诱饵来生成特定的动物了。牛是一种你发展所需要的生物。你还可以将牛放在Ex Nihilo的桶上，实现牛奶的自动收集。"
```
没有实证过，但看到过Reddit上有人发过：https://www.reddit.com/r/feedthebeast/comments/9ltc9b/undocumented_ex_nihilo_creatio_feature_automatic/